client/src/components/Header.jsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import PropTypes from 'prop-types';
```

Comment:
Why the move away from PropTypes? They seem a useful check to ensure the data is there.

Reply:
Ah yes, I'm definitely going to bring it back. I was getting all these weird errors with prop types so I removed it temporarily.